packages/cli/src/databases/entities/User.ts

```diff
@@ -115,7 +115,7 @@ export class User extends WithTimestamps implements IUser {
 	@AfterLoad()
 	@AfterUpdate()
 	computeIsPending(): void {
-		this.isPending = this.password === null;
+		this.isPending = this.password === null && this.role !== 'global:owner';
```

Comment:
Is this related? I don't get the connection to SEC-2.

Reply:
the original change to fix this issue started failing tests because in those tests we were creating instance owners without a password, but the patch endpoint was sending a password, changing the status of `isPending`. since the password wasn't being updated on that endpoint anymore (as it should), the assertions on `isPending` started failing.
We could fix those tests, but this code also seems wrong, since an instance owner can never be pending.